packages/app/src/api/mudita-center-server/client.ts

```diff
@@ -13,7 +13,12 @@ import {
 import { MuditaCenterServerRoutes } from "App/api/mudita-center-server/mudita-center-server-routes"
```

Comment:
It's looking good, but let's also think about how we can cover this functionality within our new code convention